config.go

```diff
@@ -32,7 +32,13 @@ type Config struct {
 	Project      string `yaml:"project-id"`
 	Subscription string `yaml:"subscription"`
 	// Queues is sorted by Duration, shortest first
-	Queues []Queue `yaml:"queues"`
+	Queues    []Queue   `yaml:"queues"`
+	BasicAuth BasicAuth `yaml:"basic-auth"`
+}
+
+type BasicAuth struct {
```

Comment:
I would call this Credentials,  the field name is BasicAuth which is the role of this type in BasicAuthHandler